plugins/javadoc/src/main/kotlin/javadoc/JavadocPageCreator.kt

```diff
@@ -46,12 +51,18 @@ open class JavadocPageCreator(
         ) {
             title(m.name, "0.0.1", dri = setOf(m.dri), kind = ContentKind.Main)
             list("Packages", "Package", setOf(m.dri), ContentKind.Packages, m.packages.map { p ->
-                val doc = p.documentation.entries.find { (k, _) -> k.platform == Platform.jvm }?.value?.let {
-                    it.children.joinToString("\n") { it.root.toString() }
+                val description = p.documentation.entries.find { (k, _) -> k.platform == Platform.jvm }?.value?.let {
```

Comment:
This is not intuitively easy for me to understand (because of me lacking some contextual knowledge here). Would you mind explaining this part to me?